src/include/duckdb/parser/column_definition.hpp

```diff
@@ -51,6 +57,8 @@ class ColumnDefinition {
 	void SetCompressionType(duckdb::CompressionType compression_type);
 
 	//! storage_oid
+	// the storage_oid is used to distinguish indexes in generated columns
+	// these columns do not physically exist, so the oid does not map directly to the index
```

Comment:
I appreciate the extra comments, as I agree it does need some clarification, but I'm not sure if I agree with this explanation
Could you perhaps change this to:
```suggestion
// The storage_oid is used to index columns that are physically stored, excluding generated columns.
// a INTEGER, b BOOLEAN, c GENERATED ALWAYS AS a, d VARCHAR
// 0          1          -                        2
```

Reply:
done, thx :)